confd/templates/production.tmpl

```diff
@@ -273,29 +273,76 @@
       "enlarge": "Expand timeline"
     },
     "dataInputFiltersConfig": {
-      "filters": [
-      		{
-      			"text": "REScene",
-      			"value": 1,
-      			"collapsed": true,
-				"children": [
-					{ "text": "RapidEye-1", "value": { "sensorType": "REScene", "sensorName": "RapidEye-1" } },
-					{ "text": "RapidEye-2", "value": { "sensorType": "REScene", "sensorName": "RapidEye-2" } }
-				]
-			},
-			{
-				"text": "PSScene3Band",
-                "value": 1,
-                "collapsed": true,
-            	"children": [
-					{ "text": "103e", "value": { "sensorType": "PSScene3Band", "sensorName": "103e" } },
-					{ "text": "0f21", "value": { "sensorType": "PSScene3Band", "sensorName": "0f21" } },
-					{ "text": "RapidEye-2", "value": { "sensorType": "PSScene3Band", "sensorName": "RapidEye-2" } },
-					{ "text": "1042", "value": { "sensorType": "PSScene3Band", "sensorName": "1042" } }
-            	]
+      "PLANET": {
+        "text": "Planet",
+        "value": 1,
+        "collapsed":true,
+        "children": [
+          {
+            "text":"REScene",
+            "value": {
+              "sensorType": "REScene"
+            }
+          },
+          {
+            "text": "PSScene3Band",
+            "value": {
+              "sensorType": "PSScene3Band"
+            }
+          },
+		  {
+            "text": "PSScene4Band",
+            "value": {
+              "sensorType": "PSScene4Band"
             }
-		]
+          },
```

Comment:
nit: remove the extra space here and in the rest of the places (curly braces should be aligned)